internal/timestampmeta/timestampmeta.go

```diff
@@ -0,0 +1,30 @@
+// Package timestampmeta provides utilities for preserving timestamps
+// using per-blob key-value-pairs (metadata, tags, etc.)
+package timestampmeta
```

Comment:
I think is is more of a Kopia "encoding" package where more of similar future encoding/decoding methods could live.